hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -786,6 +787,12 @@ def alter(table_name_str, wait = true, *args)
         # There are 3 possible options.
         # 1) Column family spec. Distinguished by having a NAME and no METHOD.
         method = arg.delete(METHOD)
+
+        if !method.nil? && method == 'avoid_reopening_regions'
```

Comment:
Can we just make a new argument `REOPEN_REGIONS => 'false'`

The problem I have with using METHOD is it's not clear that you can use this with other "methods" like table_unset_conf.

Reply:
Sounds great. I have already made the changes to the corresponding code.